reviewrot/basereview.py

```diff
@@ -123,6 +123,11 @@ def _call_api(self, url, method='GET', ssl_verify=True, ignore_err=False):
         try:
             response = self.get_response(method, url, ssl_verify)
             decoded_response = response.json()
+            # Some services (like pagure) return valid JSON with a 404 error.
+            # https://pagure.io/api/0/username/reponame/pull-requests
+            if not bool(response):
```

Comment:
Hm, you're right.

```python
In [1]: import requests

In [2]: response = requests.get('http://threebean.org/this-is-not-a-real-url')

In [3]: if response:
   ...:     print "seems ok"
   ...: else:
   ...:     print "it's broken!"
   ...: 
it's broken!
```

I guess I figured the "cast" to bool was explicit and would help the reader.  I can remove it though.  :)